aws_backup.tf

```diff
@@ -0,0 +1,128 @@
+# Source vault
+resource "aws_backup_vault" "source" {
+  count         = var.enabled ? 1 : 0
+  provider      = aws.source
+  name          = "${module.label.id}-source"
```

Comment:
Just a suggestion:
_Label module_ is meant to be used to get the naming right. Maybe it would be better to do all its argument references inside the generic locals.tf and then just refer to those locals instead of direct module reference. Later on, if needed, it will be much faster to change the value in local values instead of changing every module.label reference in other code. Just food for thought :-)